build/Targets/VSL.Imports.targets

```diff
@@ -413,15 +423,16 @@
 
     <!-- The below check is to decide if the assembly produced by the VSIX project should be included in the VSIX-->
     <ItemGroup>
-      <VsixInputAssembliesToSign Include="@(IntermediateAssembly)" Condition="'$(IncludeAssemblyInVSIXContainer)' == 'true'" />
+      <VsixInputAssembliesToSign Include="@(IntermediateAssembly)" Condition="'$(IncludeAssemblyInVSIXContainer)' == 'true'">
+        <StrongName>72</StrongName>
```

Comment:
It was unnecessary to set the value because if the assembly is expected to be part of the VSIX which is going to be signed then all the components will also need to be signed which requires us to set the value of `$SignAssembly` to be true, which will get the assembly assigned with the `$StrongNameCertificateFriendlyId`